internal/goldendataset/resource_generator.go

```diff
@@ -153,13 +146,11 @@ func generateFassAttributes() map[string]interface{} {
 func generateExecAttributes() map[string]interface{} {
 	attrMap := make(map[string]interface{})
 	attrMap[conventions.AttributeProcessExecutableName] = "otelcol"
-	parts := make([]otlpcommon.AnyValue, 3)
-	parts[0] = otlpcommon.AnyValue{Value: &otlpcommon.AnyValue_StringValue{StringValue: "otelcol"}}
-	parts[1] = otlpcommon.AnyValue{Value: &otlpcommon.AnyValue_StringValue{StringValue: "--config=/etc/otel-collector-config.yaml"}}
-	parts[2] = otlpcommon.AnyValue{Value: &otlpcommon.AnyValue_StringValue{StringValue: "--mem-ballast-size-mib=683"}}
-	attrMap[conventions.AttributeProcessCommandLine] = &otlpcommon.ArrayValue{
-		Values: parts,
-	}
+	parts := pdata.NewAttributeValueArray()
+	parts.ArrayVal().Append(pdata.NewAttributeValueString("otelcol"))
+	parts.ArrayVal().Append(pdata.NewAttributeValueString("--config=/etc/otel-collector-config.yaml"))
+	parts.ArrayVal().Append(pdata.NewAttributeValueString("--mem-ballast-size-mib=683"))
+	attrMap["conventions.AttributeProcessCommandLine"] = parts
```

Comment:
This is a bug, transformed from the constant `conventions.AttributeProcessCommandLine` to `"conventions.AttributeProcessCommandLine"`. Fixed in #3377.